src/FixerDefinition/VersionSpecification.php

```diff
@@ -36,19 +36,21 @@
     public function __construct($minimum = null, $maximum = null)
     {
         if (null === $minimum && null === $maximum) {
-            throw new \InvalidArgumentException('Either minimum or maximum need to be specified');
+            throw new \InvalidArgumentException('Minimum and/or maximum need to be specified.');
```

Comment:
```
Used to link alternatives.
‘a cup of tea or coffee’
‘are you coming or not?’
‘I either take taxis or walk everywhere’
‘it doesn't matter whether the theory is right or wrong’
```

by the oxford dictionaries (https://en.oxforddictionaries.com/definition/or)
2nd sample implies; you cannot come and both not come